packages/create-ripple-app/template/_example/tide/modules/event/formdata.js

```diff
@@ -0,0 +1,115 @@
+export default {
```

Comment:
What is this `_example` folder for?

Reply:
Setup a new project with an example of how to extend the config. Useful for external agency clients, but we wouldn't want to add it to internal sites.